From: Christian Kastner <debian@kvr.at>
Date: Sun, 25 May 2014 00:52:58 +0200
Subject: Use bash to execute the tests

The test fixtures are full of bashisms. Getting rid of them should be discussed
with upstream first.

Forwarded: not-needed
Last-Update: 2014-07-26
---
 tests/prepare.inc.sh | 2 +-
 tests/runtest.sh     | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

Index: pkg-keyutils/tests/prepare.inc.sh
===================================================================
--- pkg-keyutils.orig/tests/prepare.inc.sh
+++ pkg-keyutils/tests/prepare.inc.sh
@@ -4,7 +4,7 @@
 if [ "x`keyctl rdescribe @s | sed 's/.*;//'`" != "xRHTS/keyctl/$$" ]
 then
     echo "Running with session keyring RHTS/keyctl/$$"
-    exec keyctl session "RHTS/keyctl/$$" sh $0 $@ || exit 8
+    exec keyctl session "RHTS/keyctl/$$" bash $0 $@ || exit 8
 fi
 
 # Set up for the Red Hat Test System
Index: pkg-keyutils/tests/runtest.sh
===================================================================
--- pkg-keyutils.orig/tests/runtest.sh
+++ pkg-keyutils/tests/runtest.sh
@@ -13,7 +13,7 @@ fi
 for i in ${TESTS}; do
 	export TEST=$i
         cd $i
-	sh ./runtest.sh || exit 1
+	bash ./runtest.sh || exit 1
 	cd ${OLDPWD}
 done
 
